src/com/executors/executeservice/GetName.java

```diff
@@ -0,0 +1,25 @@
+package com.executors.executeservice;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.concurrent.Callable;
+
+public class GetName implements Callable<String> {
+    private Connection connect;
+    private int id;
+
+    public GetName(int id) throws Exception{
+        connect = DriverManager.getConnection("jdbc:mysql://localhost/nameBD?characterEncoding=utf8", "login", "password");
```

Comment:
It is better to use  constants for 
`"jdbc:mysql://localhost/nameBD?characterEncoding=utf8", "login", "password"`